kong/utils.go

```diff
@@ -327,6 +368,63 @@ func fillConfigRecord(schema gjson.Result, config Configuration) Configuration {
 		return true
 	})
 
+	// Filling defaults for deprecated fields
+	// Required for deck sync/diff inorder
+	// Otherwise, users keep seeing updates in these fields despite of no change
+	shortHandFields.ForEach(func(_, value gjson.Result) bool {
```

Comment:
I realize we're adding to this func here but it's already rather big and hard to comprehend.

Can we extract the added part to a named func and use it here? IMHO this would greatly enhance the readability of this section of the code.

WDYT?

Reply:
configFields are added in a similar way. So, I left it as it is to keep both of the iterators similar.
Also, there's nothing much common between the two. So, creating a different function just for one of these seemed out of the place to me.